packages/protocol/contracts/thirdparty/onchainRA/AutomataDcapV3Attestation.sol

```diff
@@ -0,0 +1,651 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
Are we using the code from the head commit of https://github.com/automata-network/automata-dcap-v3-attestation/tree/main/contracts/interfaces?

Can we try to also install it as a sub module rather than copying its code over?